k8s/namespaces/fees-pay/ccpay-payment-api/ccpay-payment-api.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccpay-payment-api
+spec:
+  releaseName: ccpay-payment-api
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/payment-api
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/payment/api:pr-972-bbd9000-20220119123414 #{"$imagepolicy": "flux-system:ithc-ccpay-payment-app"}
```

Comment:
This `image ` doesn't look correct  - if this is required; please move to ithc.yaml

Reply:
Should have added, we have various checks in place - in the main ccpay-payment-api file etc - it needs to be prod image unless specifically excluded in our checks. As mentioned, please move to ithc.yaml